src/main/java/seedu/souschef/model/ingredient/IngredientAmount.java

```diff
@@ -16,6 +16,10 @@ public IngredientAmount(String amount) throws NumberFormatException {
         this.amount = Double.parseDouble(amount);
     }
 
+    public double getAmount() {
+        return amount;
+    }
```

Comment:
This getter name is duplicated with getAmount method of other Ingredient related classes. Please remove this part.